airbyte-workers/src/test/java/io/airbyte/workers/internal/DefaultAirbyteDestinationTest.java

```diff
@@ -47,6 +47,7 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+@SuppressWarnings({"PMD.AvoidPrintStackTrace", "PMD.JUnitTestsShouldIncludeAssert"})
```

Comment:
We should probably just fix this test to use a logger instead of printing the stack trace.